redaxo/src/core/fragments/core/selects/select.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @var rex_fragment $this
+ * @psalm-scope-this rex_fragment
+ */
+$sliceSelect = new rex_select();
```

Comment:
Das hier ist ein Fragment im Core, und handelt von "Slices". Sind wir wieder bei dem Thema, dass der Core "Slices" nicht kennt, daher kann das so nicht passen.

Eigentlich kann der Code direkt mit in das "module_select.php" Fragment, statt des subfragment-Aufrufs, oder?

Reply:
>Das hier ist ein Fragment im Core, und handelt von "Slices". Sind wir wieder bei dem Thema, dass der Core "Slices" nicht kennt, daher kann das so nicht passen.

Wäre dann einfach nur `$select` passend?

>Eigentlich kann der Code direkt mit in das "module_select.php" Fragment, statt des subfragment-Aufrufs, oder?

Ich weiss nicht, weshalb das `module_select.php` das ursprünglich über das Subfragment gelöst hat, deshalb habe ich es beibehalten.
Aber grundsätzlich kann das auch alles direkt da rein, ja.